primitives/primproc/bppsendthread.h

```diff
@@ -63,7 +63,15 @@ class BPPSendThread
             msg(m), sock(so), sockLock(sl), sockIndex(si) { }
     };
 
-    bool okToProceed();
+    bool sizeTooBig()
+    {
+      // keep the queue size below the 100 msg threshold & below the 250MB mark,
+      // but at least 2 msgs so there is always 1 ready to be sent.
```

Comment:
at least 3 messages according with the condition below.

Reply:
changed the comment